src/test/java/org/academy/codewars/dataproviders/ConsoleScannerDataProvider.java

```diff
@@ -23,4 +23,20 @@ public Iterator<Object[]> dpTestReadInt() {
 
         return inputs.iterator();
     }
+
+    @DataProvider(name = "dp-testReadFloat")
+    public Iterator<Object[]> dpTestReadFloat() {
+        List<Object[]> inputs = new ArrayList<>();
+        String param = "test param";
+        String expectedOutput1 = "Enter a " + param + " (float): ";
+        String expectedOutput2 = "Enter a " + param + " (float): " +
+                "Invalid data format. A floating point number is expected.\n" +
+                "Enter a " + param + " (float): ";
+        expectedOutput2 = expectedOutput2.replaceAll("\\n|\\r\\n", System.lineSeparator());
```

Comment:
?, use /n on new line (check completed methods)

Reply:
Check screenshot in telegram